bin/install-ci-dependencies.py

```diff
@@ -173,15 +173,15 @@ def unpack(self):
 
 def get_android_url():
   if os.getenv('TRAVIS'):
-    if os.getenv('TOOLCHAIN') == 'android-ndk-r10e-api-19-armeabi-v7a-neon':
```

Comment:
Just seen the comment on #78.
Let me update the commit, so as to support both the `ndk`s.